Deployment Guide.md

```diff
@@ -38,6 +38,9 @@ $ cd cluster
 $ kubectl apply -f .
 ```
 
+8. Change the calico MTU to 1420 (This is for Jenins agents to work properly)
+   Folllow, https://cloud.ibm.com/docs/containers?topic=containers-kernel#calico-mtu and change the mtu to 1420
+
```

Comment:
I'm not really comfortable with it, will have to figure out some other way to fix it. Can be a new k8s cluster in powervs that matches MTU with IKS,?!

Reply:
Removed this part